src/App.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div id="app">
+    <!-- TODO: add routes as SPA -->
```

Comment:
I don't know much about Vue, but is this needed for the app to function?

Reply:
I took care of the SPA conversion. Technically, this whole PR is not needed for the app to function. It's just an internal task to improve developer experience.